src/database/Database.cpp

```diff
@@ -62,7 +62,7 @@ using namespace std;
 bool Database::gDriversRegistered = false;
 
 // smallest schema version supported
-static unsigned long const MIN_SCHEMA_VERSION = 13;
+static unsigned long const MIN_SCHEMA_VERSION = 20;
```

Comment:
Why is the min schema version 20 and not 21?

Reply:
The nodes will be upgrading from 20 to 21, right? In upgradeToCurrentSchema, we check if the current version is less than min version and throw an exception. After this check we call upgradeSchema(current version + 1). If `MIN_SCHEMA_VERSION = 21`, the the nodes upgrading would fail that check.

Reply:
I.e. 20 is still "supported" in that we support an upgrade path from 20 to 21.

Reply:
(Discussed offline, as there are no remaining nodes on 20 it is safe to bump the MIN_SCHEMA_VERSION in this PR)